src/storage/src/hummock/sstable/forward_sstable_iterator.rs

```diff
@@ -80,6 +86,36 @@ impl SstableIterator {
         if idx >= self.sst.value().block_count() {
             self.block_iter = None;
         } else {
+            if prefetch_num >= 1 {
+                // todo: do more prefetches
+                let prefetch_block_index = idx + 1;
+                let sst = self.sst.value();
+                let sst_id = sst.id;
+                let block_meta = sst
+                    .meta
+                    .block_metas
+                    .get(prefetch_block_index)
+                    .ok_or_else(HummockError::invalid_block)
+                    .unwrap(); // FIXME: don't unwrap here.
+                let block_loc = BlockLocation {
+                    offset: block_meta.offset as usize,
+                    size: block_meta.len as usize,
+                };
+                let uncompressed_capacity = block_meta.uncompressed_size as usize;
+                let sstable_store = self.sstable_store.clone();
+                tokio::spawn(async move {
+                    sstable_store
+                        .get_with_block_info(
```

Comment:
I prefer move this code to `SstableStore`. It means that we only need to spawn a new task to tokio-pool when the block does not exist in cache.